apps/i18n/codebridge/en_us.json

```diff
@@ -29,6 +29,7 @@
   "duplicateFolderError": "Foldername {folderName} is already in use in this folder. Please choose a different name.",
   "duplicateSupportFileError": "Filename {fileName} is already in use in this folder in the level's support code. Please choose a different name.",
   "filesHeader": "Files",
+  "fileOptions": "File Options",
```

Comment:
I see in [`aichat` locale](https://github.com/code-dot-org/code-dot-org/blob/2cc8eafedcbcba54a8f87629d1a3f564577c7e6a/apps/i18n/aichat/en_us.json#L13-L19) that the aria strings are prefixed with `aria_`. Curious if this is a convention we'd want to use in codebridge.
And is there a convention about letter case?

Reply:
I think if we could see using the string for anything else, it would make sense to not use the aria prefix, but I could be convinced! I see both caps approaches in this file ("validationResults": "Validation Results", and "testName": "Test Name" as well as strings like "verticalLayout": "Vertical layout", and  "defaultLayout": "Default layout",). Do you have a strong preference?

Reply:
Makes sense that the string could be used elsewhere!

I do not have a preference but think it'd be good to be consistent. Curious what @molly-moen thinks. Thanks!